### PR TITLE
Proof TVCEXECUTIVE #267

### PR DESCRIPTION
**Pull request by Gaboso**
Proof TVCEXECUTIVE #267

### CODE REVIEW

**Comment by chrislgarry** [comment]
@oldmud0 do you want this one? Else Ill take care of it. Thanks.

**Comment by oldmud0** [comment]
This pull request is waiting on your review

...Well, since it's clear that I am too busy with uni right now, I'll let you take care of it then. Thanks a lot.

File: Comanche055/TVCEXECUTIVE.agc [hunk 1]
```diff
@@ -59,20 +60,22 @@
...
 #
 # OTHER INTERFACES....
+#
 #      *TVCRESTART PACKAGE FOR RESTARTS
 #      *PITCHDAP, YAWDAP FOR VARIABLE GAINS AND ENGINE TRIM ANGLES
-#
 # ERASABLE INITIALIZATION REQUIRED....
+#
 #      *SEE TVCDAPON....TVCINIT4
-#      *VARK AND 1/CONACC (S40.15 OF R03)
```
Comment (chrislgarry) [inline Comanche055/TVCEXECUTIVE.agc]: R03 was way off. Thats a bit weird. Theres no R03 text in that scan. I wonder what happened.

Comment (chrislgarry) [reply Comanche055/TVCEXECUTIVE.agc]: I saw another that was way off. The transcribers must have had a bunch of papers laid out in front of them, and sometimes glanced at the wrong paper.

**Pull request merged**